src/appresolver.js

```diff
@@ -42,6 +42,7 @@ function LocalAppRegistry(appClass, opts) {
 	opts.port = opts.port || 3000;
 	opts.dist = opts.dist || 'dist';
 	opts.configFile = opts.configFile || 'appconfig.json';
+	opts.baseRoute = opts.baseRoute || '/app/';
```

Comment:
I think this should be `/app` and then down below it should be:

```js
return 'http://' + this._opts.hostname + ':' + this._opts.port + this._opts.baseRoute + '/';
```

Essentially always have the trailing slash so that they can just set baseRoute to `''` and have it work.